CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.0] - 2020-05-05
```

Comment:
Be careful with this! Version number and release date should be added right before merging the PR, not when creating it. A new PR may come in and be merged before this one, so this version and release date would not be correct

Reply:
Big true